x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/math/Greatest.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.expression.function.scalar.math;
```

Comment:
Doesn't really matter, but the package this is in is kind of an unusual choice. Is not really a great option, but if I would have to choose I'd go with `org.elasticsearch.xpack.esql.expression.function.scalar.conditional` instead.

Reply:
I think it is like `max`, I can move it.